01/17/2024 class 1

edaraa2 opened the pull request:
This file contains string operations encompasses fundamental concepts and methods essential for effective string manipulation in Python., including -Methods for iterating over strings
-Repetition and concatenation operators
-Strings as immutable objects
Repetition and concatenation operators are both used for manipulating strings in Python, but they serve different purposes:
Repetition Operator (*)
    Function: Repeats a string a specified number of times.
-Slicing strings and testing strings
Slicing lets you grab specific parts of a string like picking from a slice of pie. You use square brackets with start and end positions to extract the portion you need.

Testing strings involves checking if a substring exists within another string. Imagine searching for a word in a sentence. Python's "in" operator acts like a detective, finding the presence (or absence) of the target word.
-String methods-
String methods are like tools in a text toolbox. They help you manipulate strings in Python. Slicing lets you cut out pieces, like taking a slice of bread. Concatenation joins strings, like building a sandwich. Iteration lets you loop through each letter, like examining each ingredient. Testing with "in" checks for words, like finding cheese on your sandwich. There are many more methods for searching, replacing, and formatting text, giving you ultimate control over your strings!
-Splitting a string - Splitting a string is like dividing a pizza into slices. You use the split() method with a separator (like a comma or space) to chop the string into a list of pieces. By default, it splits on whitespace, creating a list of words. You can customize the separator to target specific characters or patterns, allowing you to break the string down into exactly the pieces you need.

edaraa2 committed:
Add files via upload

This file contains string operations encompasses fundamental concepts and methods essential for effective string manipulation in Python., including
-Methods for iterating over strings
-Repetition and concatenation operators
-Strings as immutable objects
-Slicing strings and testing strings
-String methods-
-Splitting a string

nikshepkulli commented on ClassWork 01_17.ipynb:
```diff
@@ -0,0 +1,360 @@
+{
```
<p>No headers and detailed description.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/edaraa2/Text-Based-Analysis/pull/1/discussion/#comment-1483829146'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

edaraa2 replied:
<p>Hello Professor,</p><p>Added another file with requested comments under the name of ClassWork 01.ipynb</p>

edaraa2 committed:
Add files via upload

This file contains string operations encompasses fundamental concepts and methods essential for effective string manipulation in Python., including -Methods for iterating over strings
-Repetition and concatenation operators
-Strings as immutable objects
-Slicing strings and testing strings
-String methods-
-Splitting a string
